### ISSUE TITLE
feature request - multi-threading 

### ISSUE THREAD

**Issue by james-see** [bboe/deslackify#3]
It takes a long time to delete messages one by one. It would make sense to use multithreading in python3 to make this go faster. If this is a good idea to pursue, I can work on it and submit a pull request @bboe . Cheers

**Comment by bboe** [comment]
Yeah, I think giving that a try makes sense. Just be careful about any rate limits.

**Comment by james-see** [comment]
Sure that makes sense.

**Comment by bboe** [comment]
Closing as inactive. This has been open since 2019 without a PR materializing, and Slack's API rate limits cap how much parallelism actually helps here (deletes get throttled regardless). Happy to revisit if someone wants to take a run at it. Thanks for the suggestion!
